Library/Homebrew/brew.sh

```diff
@@ -290,9 +290,22 @@ auto-update() {
       fi
     fi
 
-    # Skip auto-update if the Homebrew/brew repository has been checked in the
-    # last $HOMEBREW_AUTO_UPDATE_SECS.
     repo_fetch_head="${HOMEBREW_REPOSITORY}/.git/FETCH_HEAD"
+    if [[ -n "${HOMEBREW_AUTO_UPDATE_CORE_CLONES}" ]]
```

Comment:
```suggestion
    if [[ -n "${HOMEBREW_AUTO_UPDATE_CORE_CASK_TAPS}" ]]
```

or something? I don't think most of the time "core" naming implies "homebrew-cask"?